Dockerfile-ubi

```diff
@@ -24,12 +24,11 @@ RUN yum upgrade -y &&\
     yum remove wget tar --quiet -y &&\
     rm -rf /var/cache/dnf/*
 
-# Note: we would prefer to use our own user to ensure the image does not run in root, but this comes with a lot of
-# limitations:
-# - difficulties with docker volume mounting
-# - will not work properly in gcp cloud run (especially with data mounting)
-# TODO: revist in future if limitations are still true
-# RUN addgroup -S mondoo && adduser -S -G mondoo mondoo
-# USER mondoo
 ENTRYPOINT [ "mondoo" ]
 CMD ["help"]
+
+# Rootless version of the container
+FROM root AS rootless
+
+RUN groupadd -r mondoo && adduser -r -g mondoo mondoo
```

Comment:
Turns out, ubi also has no `groupadd` and `adduser` command.

It seems in ubi, yo just specify `USER 1001`.

https://developer.ibm.com/learningpaths/universal-application-image/design-universal-image/#2-design-the-image-to-run-as-a-non-root-user-id

Reply:
that must be only for the minimal image... I did test these things with the normal ubi image and they worked